dataactvalidator/app.py

```diff
@@ -1,111 +1,59 @@
 import logging
 
-import csv
-from flask import Flask, request, g
+import boto3
+from flask import Flask, g, current_app
 
 from dataactcore.config import CONFIG_BROKER, CONFIG_SERVICES
 from dataactcore.interfaces.db import GlobalDB
-from dataactcore.interfaces.function_bag import write_file_error, mark_job_status
 from dataactcore.logging import configure_logging
-from dataactcore.models.jobModels import Job
-from dataactcore.utils.jsonResponse import JsonResponse
-from dataactcore.utils.responseException import ResponseException
-from dataactvalidator.validation_handlers.validationError import ValidationError
 from dataactvalidator.validation_handlers.validationManager import ValidationManager
 
 
 logger = logging.getLogger(__name__)
+logging.getLogger('boto3').setLevel(logging.CRITICAL)
+logging.getLogger('botocore').setLevel(logging.CRITICAL)
 
 
 def create_app():
-    """Create the Flask app."""
-    flask_app = Flask(__name__.split('.')[0])
-    flask_app.debug = CONFIG_SERVICES['debug']
-    local = CONFIG_BROKER['local']
-    error_report_path = CONFIG_SERVICES['error_report_path']
-    flask_app.config.from_object(__name__)
-
-    # Future: Override config w/ environment variable, if set
-    flask_app.config.from_envvar('VALIDATOR_SETTINGS', silent=True)
-
-    @flask_app.teardown_appcontext
-    def teardown_appcontext(exception):
-        GlobalDB.close()
-
-    @flask_app.before_request
-    def before_request():
-        GlobalDB.db()
-
-    @flask_app.errorhandler(ResponseException)
-    def handle_response_exception(error):
```

Comment:
Any exception that gets raised will be logged, but we don't get an HTTP response any more so we don't need to explicitly handle response exceptions